### PR TITLE
added cart reducer

### PR DESCRIPTION
**Pull request by AksaCode**
I added reducer and action, also dispatched in ProductsScreen.js file. Action is tested inside the redux dev tool and it is working.

### CODE REVIEW

File: store/reducer/cardReducer.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { ADD_PRODUCT } from '../action/cart';
+
+const initState = {
+  items: [],
+  total: 0,
+};
+
+const cartReducer = (state = initState, action) => {
+  if (action.type === ADD_PRODUCT) {
```
Comment (Saky94) [inline store/reducer/cardReducer.js]: Implement reducer with switch & case instead of if/else.

File: store/action/cart.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const ADD_PRODUCT = 'ADD_PRODUCT';
```
Comment (Saky94) [inline store/action/cart.js:1]: Empty line after this.

File: screens/ProductsScreen.js [hunk 3]
```diff
@@ -1,16 +1,34 @@
...
 
 import CardWrapper from '../components/CardWrapper';
+import { addProduct } from '../store/action/cart';
 
 const ProductsList = (props) => {
   const products = useSelector((state) => state.products.products);
+  const cartItems = useSelector((state) => state.cart.items);
+  const dispatch = useDispatch();
+
+  const onAddToCart = useCallback(
```
Comment (Saky94) [inline screens/ProductsScreen.js]: I think it's not necessary to use useCallback in this case.

**Review by Saky94 — CHANGES_REQUESTED** [review]

File: screens/ProductsScreen.js [hunk 4]
```diff
@@ -1,16 +1,31 @@
-import React from 'react';
+import React, { useCallback } from 'react';
```
Comment (Saky94) [inline screens/ProductsScreen.js]: Delete unused useCallBack import.

**Review by Saky94 — APPROVED** [review]

File: screens/ProductsScreen.js (hunk 4 above)
Comment (AksaCode) [reply screens/ProductsScreen.js]: Majko mila ahahahahahah

**Pull request merged**